encrypted-media/scripts/setmediakeys-multiple-times-with-different-mediakeys.js

```diff
@@ -53,34 +53,35 @@ function runTest(config, qualifier) {
             // Set mediaKeys2 on video element (switching MediaKeys) need not
             // fail after src attribute is set.
             return _video.setMediaKeys(_mediaKeys2);
-        })).then(test.step_func(function() {
+        }).then(test.step_func(function() {
             // Switching setMediaKeys after setting src attribute on video element
```

Comment:
This seems to say the same thing as above. Instead, it could say "Switching MediaKeys succeeded..."